dj_cqrs/transport/pubsub.py

```diff
@@ -0,0 +1,383 @@
+#  Copyright © 2021 Ingram Micro Inc. All rights reserved.
```

Comment:
Thanks for you contribution.

This file seems like a custom implementation for specific project of yours. It would be nice to make this generic for the library.